llvm/CheerpCmakeConf.cmake

```diff
@@ -17,9 +17,17 @@ set(LLVM_TOOLCHAIN_TOOLS
   llc
   CACHE STRING "")
 
-set(LLVM_DISTRIBUTION_COMPONENTS
-  clang
-  clang-resource-headers
-  ${LLVM_TOOLCHAIN_TOOLS}
-  CACHE STRING "")
-
+if("${LLVM_ENABLE_PROJECTS}" MATCHES ".*clang-tools-extra.*")
```

Comment:
change this so that we only set LLVM_DISTRIBUTION_COMPONENTS once, with a variable ${CLANGD} that might be empty depending on this condition.